modules/sdk-api/test/unit/v1/wallet.ts

```diff
@@ -358,6 +358,34 @@ describe('Wallet Prototype Methods', function () {
       );
     });
 
+    it('signs an unsigned tx made of uncompressed public keys and verifies signatures', async function () {
+      const unsignedTxHex =
+        '020000000208d1499ff2cca5c81d5a95dbc55d9f85ed3a023719335a04569387ec9c87fb480000000000fffffffffee508e887d01f8ee77444ec28f1c85f461e1a71fefb0196d11b708bdc115da50000000000ffffffff01020a03000000000017a9144224f4bfbcacfe36c211a681456b19ae8270b7078700000000';
```

Comment:
to avoid long lines, you should move these values to a JSON file and load it here

the json file should explain how you obtain the values in `comment` property